internal/interactive/print.go

```diff
@@ -55,7 +55,10 @@ func Indent(str string, indent int) string {
 	lines := strings.Split(str, "\n")
 	result := ""
 	for _, line := range lines {
-		result += padding + line + "\n"
+		if line != "" {
```

Comment:
should it avoid `+ "\n"` when line is empty ?

Reply:
When working with a paragraph you may want to include empty line separator
```
  this is a line
<-- no space at the beginning of this line
  this is a line2
```